apps/passport-client/src/localstorage.ts

```diff
@@ -33,6 +34,7 @@ export async function loadPCDs() {
       SemaphoreGroupPCDPackage,
       SemaphoreIdentityPCDPackage,
       SemaphoreSignaturePCDPackage,
+      RLNPCDPackage,
```

Comment:
Would we also need 
```typescript
  await RLNPCDPackage.init({
    wasmFilePath: SERVER_STATIC_URL + "/rln-artifacts/16.wasm",
    zkeyFilePath: SERVER_STATIC_URL + "/rln-artifacts/16.zkey",
  });
```
in `loadPCDs` or it can be done elsewhere?

Reply:
You are correct. I've actually just merged a PR which slightly changes the location where this happens: https://github.com/proofcarryingdata/zupass/pull/223. You'll need to update it there.

Reply:
Thanks for the direction. I just did it in https://github.com/proofcarryingdata/zupass/pull/161/commits/f41b103ac11d596ccd3023417c74a855ff5dbe45, but I'm not sure if I missed anything. Could you confirm if it's done correctly? Thanks 🙏